### ISSUE TITLE
Callback for button release?

### ISSUE THREAD

**Issue by mrdc** [evert-arias/EasyButton#30]
Hello,

Would be nice to add a callback for button release.

Thanks!

**Comment by elC0mpa** [comment]
There is a callback for button release, is onPressed event, maybe what you mean is that it would be nice to have a onReleasedFor event. Thanks for your feedback

**Comment by mrdc** [comment]
Yes, just for release event. I need to catch once when there is HIGH on pin, do some work and catch once LOW on pin. What I have now is that my Serial is flooded if I use wasPressed and wasReleased.

**Comment by source-creator** [comment]
Greetings @elC0mpa 
The callback is not called for me when the button is released, only when it is pressed.
Perhaps I am missing something?

**Comment by evert-arias** [comment]
Hi @source-creator 
Please provide your code to see if there is any issue and we'll try to reproduce it in our side.

**Comment by gogreenpower** [comment]
Did anyone get a OnRelease Callback working?

**Comment by eppfel** [comment]
I setup a check for `btn.isReleased()` after `btn.read()` once `pressedFor(X)`was triggered.

```
btn.read();
if (waspressedforsometime && btn.isReleased())
{
  onRelease();
}
```
